### ISSUE TITLE
'NoneType' object has no attribute 'get'

### ISSUE THREAD

**Issue by Thomasorus** [iamtalhaasghar/yewtube#5]
Hey, it's me again, sorry for suddenly finding bugs.

## Issue / Suggestion

When using `//` or `..` to search for a playlist, yewtube returns:

```
Error fetching data. Possible network issue.
'NoneType' object has no attribute 'get'
```

## Your Environment
Music player: mpv
Version:

```
yewtube version      : 1.1.0
   notes           : released on 28 Jan 2022
youtube-dl version : 2021.12.17
Python version     : 3.9.10 (main, Jan 15 2022, 11:40:53)
[Clang 13.0.0 (clang-1300.0.29.3)]
Processor          : arm
Machine type       : arm64
Architecture       : 64bit,
Platform           : macOS-12.1-arm64-arm-64bit
sys.stdout.enc     : utf-8
default enc        : utf-8
Config dir         : /Users/thomasperso/.config/mps-youtube
env:TERM           : xterm-256color
env:SHELL          : /bin/zsh
env:LANG           : fr_FR.UTF-8
```

**Comment by iamtalhaasghar** [comment]
This is an expected behavior as yewtube is in transition period of completely getting rid of youtube api v3.

**Comment by iamtalhaasghar** [comment]
fixed in 5490c240e42905fcd202ef78a9d975d5e5554d4f. Upgrade and enjoy!
